hbase-examples/pom.xml

```diff
@@ -200,11 +200,6 @@
       <artifactId>log4j-slf4j-impl</artifactId>
       <scope>test</scope>
     </dependency>
-    <dependency>
```

Comment:
Actually, this one should be kept, and the other one should be removed, so that that hte logging backend jards are grouped together.